src/components/common/FormInputError.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { FormInputError, Props } from 'components/common/FormInputError'
+
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Meta, Story } from '@storybook/react/types-6-0'
+
+export default {
+  title: 'Common/FormInputError',
```

Comment:
Why you don't provide a `Component`?

Reply:
No reason, cause it worked without it. Now I read more carefully I see:

> The component field is optional (but encouraged!), and is used by addons for automatic prop table generation and display of other component metadata. title should be unique, i.e. not re-used across files.